frontend/js/app/nginx/access/form.ejs

```diff
@@ -60,10 +66,19 @@
                     </div>
 
                     <div class="items"><!-- items --></div>
+                    <div class="btn-list justify-content-end">
+                        <button type="button" class="btn btn-teal auth_add"><%- i18n('access-lists', 'auth-add') %></button>
+                    </div>
                 </div>
 
                 <!-- Access -->
                 <div class="tab-pane" id="access">
+                    <p>
+                        IP Address Whitelist/Blacklist via
+                        <a href="https://nginx.org/en/docs/http/ngx_http_access_module.html">
```

Comment:
Can you please add `target="_blank"` to prevent losing any data currently entered